speedtracker/benchmarks/canonicalExampleBenchmark.m

```diff
@@ -2,7 +2,7 @@
 %CANONICALEXAMPLEBENCHMARK simple benchmark using the same RHS and settings as the Readme example
     options = odeset('AbsTol', 1e-8, 'RelTol', 1e-6);
     datahandle = prepareDatahandleForIntegration('benchmark_canonicalExampleRHS', ...
-                                                 'solver', 'ode45', ...
+                                                 'integrator', 'ode45', ...
```

Comment:
Pretty sure the name of the integrator parameter is still 'solver' not 'integrator'. We wanted to change it to 'integrator' at some point, but before this change happens stick to using 'solver'. Otherwise, IFDIFF will just ignore your 'integrator' parameter silently and use a default value (ode15s) for the integrator.